setup.py

```diff
@@ -17,6 +17,7 @@
     package_data={
         "web_poet": ["py.typed"],
     },
+    entry_points={"pytest11": ["web-poet = web_poet.testing.pytest"]},
```

Comment:
Should we also include `pytest` as a dependency?

Reply:
I don't know, I even thought whether we should split this into a separate pytest-web-poet package to avoid this (which is probably too much work).

Reply:
So, I think this is not needed, as if you want to use the plugin you should already have pytest installed, and if you don't need it you can import even `web_poet.testing` without installing pytest.